src/nvim/eval.c

```diff
@@ -7076,6 +7076,70 @@ static void f_assert_true(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   assert_bool(argvars, true);
 }
 
+/// "async_invoke(callee, args, opts[, callback])" function
+static void f_async_invoke(typval_T *argvars, typval_T *rettv, FunPtr fptr)
```

Comment:
Since there is already a `call()` function (yes, `:call call(...)`) I guess it makes sense to name this `call_async()`.